### DISCUSSION TITLE
.Net SDK

### DISCUSSION THREAD

**Discussion by brook-miller** [deepgram/community#603] [General help] [answered]
The .net package from nuget seems to be a very different api than the one that is documented and hosted on github.  does anyone have updated example code to use the .net API?  

```
 var util = new HttpClientUtil(); //as far as I can tell this class is internal only so one cannot actually create an instance to pass as a required parameter.

 var deepgramClient = new PrerecordedTranscriptionClient(new Credentials("apikey"), util);

 var options = new PrerecordedTranscriptionOptions()
 {
     Punctuate = true

 };
 var response = new PrerecordedTranscription();
 using (var stream = new FileStream(filepath, FileMode.Open))
 {
     var source = new StreamSource(stream, "audio/wav");

     response = await deepgramClient.GetTranscriptionAsync(source, options);
 }

```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
Hi @brook-miller  I just pushed a test build for the upcoming `v4` release but it should have been under the `Deepgram.NET.SDK` development package. The stable version should be under the `Deepgram` package in Nuget. Can you please check which package dependency you are referencing?

**Comment by davidvonthenen** [comment, marked as answer]
@brook-miller you should find the correct nuget package here:
https://www.nuget.org/packages/Deepgram/3.4.1

To avoid confusion, I have disabled the poorly named Developer package to make clear that these are dev builds that shouldn't be installed.
